### PR TITLE
Sprint 33 TLT-2066 Add support for multiple mailing list addresses in the To field

### PR DESCRIPTION
**Pull request by douglashall**

### CODE REVIEW

**Comment by sapnamysore** [comment]
@sapnamysore

File: mailgun/route_handlers.py [hunk 1]
```diff
@@ -42,162 +42,159 @@ def handle_mailing_list_email_route(request):
...
-    recipient = request.POST.get('recipient')
+    recipients = set(address.parse_list(request.POST.get('recipient')))
     subject = request.POST.get('subject')
     body_plain = request.POST.get('body-plain', '')
     body_html = request.POST.get('body-html', '')
     message_id = request.POST.get('Message-Id')
-    to_list = address.parse_list(request.POST.get('To'))
-    cc_list = address.parse_list(request.POST.get('Cc'))
+    to_list = request.POST.get('To')
+    cc_list = request.POST.get('Cc')
```
Comment (rascalking) [inline mailgun/route_handlers.py]: These need to be a list of address objects, so we can construct `recipient_variables` during the POST to mailgun, telling them not to include the whole set of people we're mailing to in the To/CC field.

Comment (rascalking) [inline mailgun/route_handlers.py]: See https://github.com/Harvard-University-iCommons/lti_emailer/blob/develop/mailgun/listserv_client.py#L66

Comment (douglashall) [inline mailgun/route_handlers.py]: List of strings, yes. Fixing.

Comment (sapnamysore) [inline mailgun/route_handlers.py]: I was going to check about this change as well. Thx for the link

File: mailgun/route_handlers.py [hunk 2]
```diff
@@ -42,162 +42,159 @@ def handle_mailing_list_email_route(request):
...
-            for f in inlines:
-                logger.debug(u'Replacing "%s" with "%s" in body', f.cid, f.name)
-                body_plain = re.sub(f.cid, f.name, body_plain)
-                body_html = re.sub(f.cid, f.name, body_html)
-
-        # convert the original to/cc fields back to strings so we can send
-        # them along through the listserv
-        to_list = [a.full_spec() for a in to_list]
-        cc_list = [a.full_spec() for a in cc_list]
-
```
Comment (sapnamysore) [inline mailgun/route_handlers.py]: Presume you will add this back as well

File: mailgun/route_handlers.py (hunk 1 above)
Comment (rascalking) [inline mailgun/route_handlers.py]: Gah...never mind.  Those are getting passed in as original_to_address/original_cc_address, they can totally be left alone.  Sorry for the false alarm.

**Comment by sapnamysore** [comment]
Would be good to have a unit test with multiple recipients..Other than that , change looks good! :+1:

**Comment by douglashall** [comment]
Added regression test for multiple mailing list recipients.

**Comment by douglashall** [comment]
Closing this so I can merge to release/v1.5.1

**Pull request closed without merging**